src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <button
+              type="button"
+              onClick={() => {
+                if (tab.id !== activeTabId) {
+                  onTabSelected(tab.id);
+                }
+              }}
+            >
+              <li
+                className={tab.id === activeTab.id ? 'is-active' : ''}
+                key={tab.id}
+                data-cy="Tab"
+              >
+                <a href={`#${tab.id}`} data-cy="TabLink">
+                  {tab.title}
+                </a>
+              </li>
+            </button>
```

Comment:
Using a `<button>` element to wrap a `<li>` is not semantically correct and can cause accessibility issues. Consider restructuring this to ensure proper semantic HTML. For example, you could place the button inside the list item instead.